### PR TITLE
git ship: comment out default commit message

### LINKED ISSUE #382: git ship: Disallow the default commit message
When shipping, the default commit message almost never makes sense. When using Git Town, it often reads like

```
Squashed commit of the following:
commit 1c0c837520d5671b59130bbfd8a2020dea58e20e
Merge: e4a56b3 4be83e3
Author: Nir Shtuhl <shtuhl.n@husky.neu.edu>
Date:   Wed Feb 18 11:25:40 2015 -0800

    Merge branch 'master' into change-docs-to-biscotto

commit e4a56b3
Author: Nir Shtuhl <shtuhl.n@husky.neu.edu>
Date:   Wed Feb 18 10:13:36 2015 -0800

    update documentation to Biscotto
```

Git Town should reject the ship if the default commit message is provided.

### PR DESCRIPTION
**Pull request by charlierudolph**
@kevgo @allewun 

resolves #382

### CODE REVIEW

File: features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature [hunk 1]
```diff
@@ -0,0 +1,46 @@
...
+      | feature | git merge --no-edit origin/feature |
+      |         | git merge --no-edit main           |
+      |         | git checkout main                  |
+      | main    | git merge --squash feature         |
+      |         | git commit                         |
+      |         | git reset --hard                   |
+      |         | git checkout feature               |
+      | feature | git checkout main                  |
+      | main    | git checkout feature               |
+    And I get the error "Aborting ship due to empty commit message"
```
Comment (kevgo) [inline features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature]: Is the message really empty here, or is it the default message!

File: features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature [hunk 2]
```diff
@@ -0,0 +1,46 @@
...
+    And I am still on the "feature" branch
+    And I still have the following commits
+      | BRANCH  | LOCATION | MESSAGE        | FILE NAME    | FILE CONTENT    |
+      | feature | local    | feature commit | feature_file | feature content |
+
+
+  Scenario: undo
```
Comment (kevgo) [inline features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature]: Should we check --abort as well here?

**Comment by kevgo** [comment]
Yay, this is awesome! We needed this for a long time!  :ship:

File: features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature (hunk 1 above)
Comment (charlierudolph) [inline features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature]: The message is commented out which means by committing without any changes, the commit message is empty.

File: features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature (hunk 2 above)
Comment (charlierudolph) [inline features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature]: This should probably be moved to shared. Checking that you can't undo after an unsuccessful command

File: features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature (hunk 1 above)
Comment (kevgo) [inline features/git-ship/current_branch/on_feature_branch/without_open_changes/default_commit_message.feature]: Ah, cool

**Pull request closed without merging**